### PR TITLE
update gdp6h example for new netcdf

### PR DESCRIPTION
**Pull request by selipot**
I am expecting @philippemiron to review and add an `order_by_start_date` function

### CODE REVIEW

**Review by philippemiron — COMMENTED** [review]
We (at least) forgot to include `start_date`, `start_lon`, `start_lat`. Do you have the complete list of variables that were added?

**Comment by selipot** [comment]
The start variables are listed as metadata in the 6-hourly notebook example but not in the hourly example you are right. That is fine because the updated hourly files are not available yet. That's it, only 3 variables added: start_[date,lat,lon]

**Comment by selipot** [comment]
I am looking at `gdp-6hourly.ipynb` and `gdp6h.py`. I think we should modify this example to be able to set lon, lat, and ids as data variables instead of coordinates. Right now this does not work when exporting to parquet as an example. In addition, we should be able to rename the variable longitude and latitude when turning them into data variables, this is currently not possible.

File: data/gdp.py [hunk 1]
```diff
@@ -11,11 +11,75 @@
 from dataclasses import dataclass
 import warnings
 
-folder = "../data/raw/gdp-v2.00/"
+
+def parse_directory_file(filename: str) -> pd.DataFrame:
```
Comment (milancurcic) [inline data/gdp.py:15]: This function seems exactly the same in both gdp.py and gdp6h.py. Can we define it in its own module and import it from gdp and gdp6h modules?

File: data/gdp.py [hunk 2]
```diff
@@ -11,11 +11,75 @@
...
+
+    Returns:
+        pd.DataFrame: sorted list of drifters
+    """
+    aoml_dirfl_url = "https://www.aoml.noaa.gov/ftp/pub/phod/buoydata/"
+
+    df = pd.read_csv(join(aoml_dirfl_url, filename), delimiter="\s+", header=None)
+    df[4] = df[4] + " " + df[5]
+    df[8] = df[8] + " " + df[9]
+    df[12] = df[12] + " " + df[13]
```
Comment (milancurcic) [inline data/gdp.py:32]:
```suggestion
    df[4] += " " + df[5]
    df[8] += " " + df[9]
    df[12] += " " + df[13]
```

Can you also write a short comment above these three lines explaining why we do this?

File: data/gdp.py [hunk 3]
```diff
@@ -11,11 +11,75 @@
 from dataclasses import dataclass
 import warnings
 
-folder = "../data/raw/gdp-v2.00/"
+
+def parse_directory_file(filename: str) -> pd.DataFrame:
+    """
+    Read a directory file
```
Comment (milancurcic) [inline data/gdp.py:17]: What's a directory file?

**Review by milancurcic — APPROVED** [review]

File: data/gdp.py (hunk 3 above)
Comment (selipot) [reply data/gdp.py:17]: The directory files are metadata files maintained by the DAC.

File: data/gdp.py (hunk 1 above)
Comment (selipot) [reply data/gdp.py:15]: We can but not crucial

**Comment by selipot** [comment]
> I am looking at `gdp-6hourly.ipynb` and `gdp6h.py`. I think we should modify this example to be able to set lon, lat, and ids as data variables instead of coordinates. Right now this does not work when exporting to parquet as an example. In addition, we should be able to rename the variable longitude and latitude when turning them into data variables, this is currently not possible.

But we still cannot set lon, lat, and ids as data variables? And rename variables? This is beyond my python abilities.

File: data/gdp.py (hunk 3 above)
Comment (selipot) [reply data/gdp.py:17]: That's an AOML metadata file

**Pull request merged**